riscv-target/opentitan/device/rv32imc/link.ld

```diff
@@ -0,0 +1,55 @@
+/* Copyright lowRISC contributors.
+   Licensed under the Apache License, Version 2.0, see LICENSE for details.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+OUTPUT_ARCH( "riscv" )
+ENTRY(_start)
+
+/* required to correctly link newlib */
+GROUP( -lc -lgloss -lgcc -lsupc++ )
+
+SEARCH_DIR(.)
+__DYNAMIC  =  0;
+
+MEMORY
+{
+    flash (rx)     : ORIGIN = 0x20000000, LENGTH = 0x100000
+    ram (wx)      : ORIGIN = 0x10000000, LENGTH = 0x10000
+}
+
+_stack_start = ORIGIN(ram) + LENGTH(ram);
+
+/* need to move signature data to modifiable memory */
+SECTIONS
+{
+  .text.init ORIGIN(flash) : {
+             *(.text.init)
+  } > flash
+
+  .text.trap : {
+        . = ALIGN(0x100);
+        *(.text.trap)
+  } > flash
```

Comment:
Fix indentation.

Reply:
i'm not exactly sure what's expected here, so i just put 2 spaces for all of them.